Test/gsapTests/PredictorTests.cpp

```diff
@@ -1,19 +1,4 @@
-/**  PredictorTests - Body
- *   @file      Unit tests for Predictor classes
- *   @ingroup   GPIC++
- *   @ingroup	Predictors
- *
- *   @brief     Unit tests for Predictor classes
- *
- *   @author    Matthew Daigle
- *   @version   1.1.0
- *
- *   @pre       N/A
- *
- *      Contact: Matthew Daigle (matthew.j.daigle@nasa.gov)
- *      Created: March 25, 2016
- *
- *   @copyright Copyright (c) 2018 United States Government as represented by
+/**  Copyright (c) 2018 United States Government as represented by
```

Comment:
This should really be a regular comment, not a doc comment. Doesn't really matter since we don't do anything with doc comments right now anyway.